Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : Date {get}
+    var eventDescription: String {get} //NOT related to event content (e.g. workshop content)
```

Comment:
I find the naming/documentation of this variable a tad confusing. On first read, I thought that the "workshop content" was suitable for this but anything to do with the event wasn't.

As it is the responsibility of `Event` implementations to provide this, is this detail something that's more important for `WorkshopEvent`?

I'm not sure if a `String` can represent the information required here (e.g. rich text w/ markdown). Workshops use this [`Text`](https://github.com/hackersatcambridge/hac-website/blob/master/Sources/HaCWebsiteLib/Text.swift) type.

Finally, how long is this description meant to be? `PostCards` require a few sentences, but an event page will likely have more detailed text. Do we need multiple descriptions to handle these cases?

Reply:
@Pinpickle So I think here what we could have is a `tagline` which would simply be a `String`. 

I think any sort of other `Description` probably needs to go in the implementation because not all events may need this. E.g. a hackathon would have its own separately developed page rather than a generated page (I imagine) and therefore I can't see where we would actually surface this description??